unsubscribes_test.go

```diff
@@ -1,33 +1,31 @@
-package mailgun
+package mailgun_test
```

Comment:
please keep the package name the same.  Only one package should exist per directory and that also makes it inconsistent with the rest of the code

Reply:
> Only one package should exist per directory

it doesn't relate to `_test`, TBH.

Having `_test` is more about Black box testing - https://medium.com/tech-at-wildlife-studios/testing-golang-code-our-approach-at-wildlife-6f41e489ff36

Reply:
@tsidei, so the question is more about do we need a White box or Black box testing here.

Reply:
oh yeah, that's true. Thanks @vtopc, forgot about that. Nevermind @hirokisan , all good here.